specs/SUMMARY.md

```diff
@@ -36,14 +36,14 @@
   - [System Config](./protocol/system_config.md)
   - [Configurability](./protocol/configurability.md)
 - [Experimental]()
-  - [Fault Proof](./experimental/fault-proof/index.md)
-    - [Cannon Fault Proof VM](./experimental/fault-proof/cannon-fault-proof-vm.md)
-    - [Stage One Decentralization](./experimental/fault-proof/stage-one/index.md)
-      - [Dispute Game Interface](./experimental/fault-proof/stage-one/dispute-game-interface.md)
-      - [Fault Dispute Game](./experimental/fault-proof/stage-one/fault-dispute-game.md)
-        - [Honest Challenger](./experimental/fault-proof/stage-one/honest-challenger-fdg.md)
-        - [Bond Incentives](./experimental/fault-proof/stage-one/bond-incentives.md)
-      - [Bridge Integration](./experimental/fault-proof/stage-one/bridge-integration.md)
+  - [Fault Proof](fault-proof/index.md)
```

Comment:
Looks like this is still nested under `[Experimental]()`?

Reply:
Good spot, I didn't even know that list was there (GoLand just updated all the URLs for me).  Fixed now.